tools/install_so_files.sh

```diff
@@ -0,0 +1,9 @@
+bazel build //tensorflow_addons/...
+cp ./bazel-bin/tensorflow_addons/custom_ops/activations/_activation_ops.so ./tensorflow_addons/custom_ops/activations/
+cp ./bazel-bin/tensorflow_addons/custom_ops/image/_distort_image_ops.so ./tensorflow_addons/custom_ops/image/
+cp ./bazel-bin/tensorflow_addons/custom_ops/layers/_correlation_cost_ops.so ./tensorflow_addons/custom_ops/layers/
+cp ./bazel-bin/tensorflow_addons/custom_ops/text/_parse_time_op.so ./tensorflow_addons/custom_ops/text/
+cp ./bazel-bin/tensorflow_addons/custom_ops/seq2seq/_beam_search_ops.so ./tensorflow_addons/custom_ops/seq2seq/
+cp ./bazel-bin/tensorflow_addons/custom_ops/image/_image_ops.so ./tensorflow_addons/custom_ops/image/
+cp ./bazel-bin/tensorflow_addons/custom_ops/image/_resampler_ops.so ./tensorflow_addons/custom_ops/image/
+cp ./bazel-bin/tensorflow_addons/custom_ops/text/_skip_gram_ops.so ./tensorflow_addons/custom_ops/text/
```

Comment:
Hmmm anyway we can automate this? I don't think there should be other .so files other than the ones we want to copy?

Reply:
I'd like to automate this but I have no idea how to. Maybe we can merge this and open an issue? Maybe someone else has a good idea.

Reply:
I used a precise wildcard pattern because if not, we end up with other so like `tensorflow_addons/custom_ops/activations/lib_activation_ops_gpu.so`.  Are those useful?